app/src/main/java/com/codingblocks/cbonlineapp/mycourse/MyCourseViewModel.kt

```diff
@@ -49,6 +50,7 @@ class MyCourseViewModel(
     var name by savedStateValue<String>(handle, COURSE_NAME)
     var runId by savedStateValue<String>(handle, RUN_ID)
     var premiumRun by savedStateValue<Boolean>(handle, PREMIUM)
+    var courseId: String?= null
```

Comment:
why this isn't being saved to saved Instance  ?

Reply:
Done